templates/package/content/maven.tmpl

```diff
@@ -0,0 +1,71 @@
+{{if eq .PackageDescriptor.Package.Type "maven"}}
+	<h4 class="ui top attached header">{{.i18n.Tr "packages.installation"}}</h4>
```

Comment:
Maven repositories are not only usable with Maven itself, but also with Gradle, sbt, Leiningen... It would be nive if there could be a similar screen to the one of [Jitpack](https://jitpack.io) that makes it easier to copy-paste the repos and deps to the build files.

View on Jitpack:
![Screenshot 2021-12-06 at 16-12-19 JitPack Publish JVM and Android libraries](https://user-images.githubusercontent.com/80460567/144871594-b4fe7d7b-2e19-4423-b78a-c0b46ed8c746.png)